### ISSUE TITLE
Canvas does not have accessible name. Does not accomodate keyboard navigation.

### ISSUE THREAD

**Issue by muckee** [tsparticles/tsparticles#4785]
### What do you need?

The react-particles package fails accessibility tests because the canvas has no accessible name by default and because it does not support keyboard navigation.

In order to reproduce, simply render any react-particles element and then lint with andi.js or similar.

The name could perhaps be derived from the ID, although this solution may be inadequate as IDs are not typically human-readable and any RegEx that might be applied would be inconsistent depending on how the user chooses to format IDs.

In order to resolve the keyboard navigation issue, I suggest that canvas elements should have the property `aria-hidden='true'` and/or `role="presentation"` / `role="none"` by default.

### tsParticles Version

^2.3.4

### Which library are you using?

React.js (react-particles, react-tsparticles, react-particles-js)

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct
